core/ui/src/main/java/com/mifos/core/ui/theme/Theme.kt

```diff
@@ -0,0 +1,52 @@
+package com.mifos.core.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.darkColorScheme
+import androidx.compose.material3.lightColorScheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import com.mifos.core.designsystem.theme.Black
+import com.mifos.core.designsystem.theme.BluePrimary
+import com.mifos.core.designsystem.theme.BluePrimaryDark
+import com.mifos.core.designsystem.theme.BlueSecondary
+
+private val LightThemeColors = lightColorScheme(
```

Comment:
Theme should be part of designsystem. take reference of wallet app

Reply:
Done